pkg/man/docflags.go

```diff
@@ -0,0 +1,23 @@
+package man
+
+import "fmt"
+
+type DocFlag struct {
+	Name        string `yaml:"name"`
+	Description string `yaml:"description"`
+	Shorthand   string `yaml:"shorthand"`
+	Default     string `yaml:"default"`
```

Comment:
are we not actually consuming the data type from the yaml (i.e.: https://github.com/opentdf/otdfctl/pull/76/files#diff-cd82de12ea056f2f9204e40d24c61a91322da779130e06d1605d373447c29fc2R29)? If so, I think we should remove it.